src/libstd/collections/hash/set.rs

```diff
@@ -193,6 +193,13 @@ impl<T, S> HashSet<T, S>
         }
     }
 
+    /// Returns a reference to the underlying HashMaps hasher.
```

Comment:
I would just change this to "the set's hasher"; the fact that sets are "just maps" is an impl detail.